RollbarTests/PayloadTruncationTests.m

```diff
@@ -158,24 +140,12 @@ - (void)testErrorReportingWithTruncation {
                      data:@{@"extra_truncatable_data": simulatedLongString}
          ];
 
-        [NSThread sleepForTimeInterval:5.0f];
+        // What is this doing?
```

Comment:
These can be removed. I sometimes piggy back on existing unit tests to quickly verify newly added functionality and in this case forgot to clean it up. Thanks for catching it!